src/js/L.DNC.js

```diff
@@ -11,7 +11,22 @@
         window.L.DNC = DNC;
 
         L.DNC.init = function(){
-            L.DNC.mapView = new L.DNC.MapView();
+
+            this.mapView = new L.DNC.MapView();
+            this.dropzone = new L.DNC.DropZone( this.mapView._map, {} );
+            this.layerlist = new L.DNC.LayerList( { layerContainerId: 'dropzone' } ).addTo( this.mapView._map );
+            this.geoMenu = new L.DNC.Menu( this.layerlist, {} );
+
+            // examples of events that L.DNC.DropZone.FileReader throws
+            this.dropzone.fileReader.on( "fileparsed", function(e){
+                // TODO: This should be refactored so that this.dropzone and
+                // this.layerlist are not so tightly coupled. The logic behind
+                // this tooling should exist within their respective modules.
+                console.debug( "[ FILEREADER ]: file parsed > ", e.file );
+                var mapLayer = L.mapbox.featureLayer(e.file);
+                this.layerlist.addLayerToList( mapLayer, e.fileInfo.name, true );
+                this.numLayers++;
+            }.bind(this));
```

Comment:
@svmatthews I agree that putting this `fileReader` event handler into `init` feels wrong.  I originally pulled it out of `L.DNC.MapView` which felt more wrong.  The reason that I didn't move it into the `fileReader` (as was done with the `this.dropzone.fileReader.on( "enabled",...` handler that was in `MapView`) was that the handler currently requires access to `L.DNC.layerlist` and `L.DNC.numLayers`.  Rather than embed tightly coupled code deep into `dropzone`, I put the `TODO` on line 22 and figured we'd resolve it later through some better event handling (which is something that I don't really know much about ATM).

I could move it, but worried that it would be "out of sight, out of mine" in the `fileReader` module rather than an obvious need for refactoring within `init`.  Feel free to make the call, it's kind of six vs half dozen.